use-timescale/vpc.md

```diff
@@ -26,7 +26,8 @@ Your apps run inside your AWS Customer VPC, your Timescale Services always run
 inside the secure Timescale VPC. You control secure communication between apps in
 your VPC and your Timescale Services using a dedicated Peering VPC. The AWS PrivateLink connecting 
 Timescale VPC to the dedicated Peering VPC gives the same level of protection as using a direct AWS PrivateLink connection. 
-It only enables communication between your Customer VPC and your Timescale Services.
+It only enables communication to be initiated from your Customer VPC to Timescale Services running
```

Comment:
```suggestion
This enables you to initiate unidirectional communication from your Customer VPC to Timescale Services running
```

Reply:
@billy-the-fish the suggestion leads to confusion. The communication will go in both directions (you make requests and get data back). But the communication can only be initiated (i.e. opening the connection) from the Customer VPC.